api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/BaseDaoImpl.java

```diff
@@ -792,9 +804,8 @@ protected Criterion generateSystemQuery(String system, List<String> codes) {
 		}
 	}
 	
-	protected void handleResourceCode(Criteria criteria, TokenOrListParam code, @NotNull String conceptAlias,
+	protected void handleCode(Criteria criteria, TokenOrListParam code, @NotNull String conceptAlias,
```

Comment:
Can this also be called `handleCodeableConcept()`? Also, can we move it so it's with the other `handleCodeableConcept` method?